src/libs/NumberUtils.js

```diff
@@ -0,0 +1,26 @@
+import CONST from "../CONST";
+
+/**
+ * Generates a random positive 64 bit integer by randomly generating the left half and right half and concatenating them
+ * @returns {string} randomly generated 64 bit string
+ */
+function rand64() {
+    // Max 64-bit signed:
+    // 9,223,372,036,854,775,807
+    // The top part of the max 64-bit number *+1* because we're flooring it
+    let top = Math.floor(Math.random() * CONST.MAX_64BIT_TOP_HALF + 1);
+  
+    // The bottom part of the max 64-bit number *+1* for the same reason
+    let rest = Math.floor(Math.random() * CONST.MAX_64BIT_BOTTOM_HALF + 1);
```

Comment:
Let's either make this into a ternary, or an else below so that we don't have to generate a number twice for any number that doesn't start with the max first half